php/libraries/Utility.class.inc

```diff
@@ -28,7 +28,7 @@ class Utility
     /**
      * Stops anyone from instantiating an instance of Utility
      */
-    private function __construct()
+    public function __construct()
```

Comment:
There's no reason to make the Utility class constructable. It's holds a bunch of static functions, there is no instance data to go with it.

Reply:
Hi! Sorry about that I misunderstood how the Utility class was organized and how the methods were to be called. Just pushed my edits!